[2025-01-24T21:23:17.966+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Train.update_model manual__2025-01-24T21:23:01.338798+00:00 [queued]>
[2025-01-24T21:23:17.981+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Train.update_model manual__2025-01-24T21:23:01.338798+00:00 [queued]>
[2025-01-24T21:23:17.982+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-01-24T21:23:18.004+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): update_model> on 2025-01-24 21:23:01.338798+00:00
[2025-01-24T21:23:18.014+0000] {standard_task_runner.py:60} INFO - Started process 158 to run task
[2025-01-24T21:23:18.019+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Train', 'update_model', 'manual__2025-01-24T21:23:01.338798+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmprf7mt7ng']
[2025-01-24T21:23:18.022+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask update_model
[2025-01-24T21:23:18.048+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-01-24T21:23:18.113+0000] {task_command.py:423} INFO - Running <TaskInstance: Train.update_model manual__2025-01-24T21:23:01.338798+00:00 [running]> on host 08a333b8a8e7
[2025-01-24T21:23:18.205+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='Train' AIRFLOW_CTX_TASK_ID='update_model' AIRFLOW_CTX_EXECUTION_DATE='2025-01-24T21:23:01.338798+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-24T21:23:01.338798+00:00'
[2025-01-24T21:23:18.743+0000] {logging_mixin.py:188} INFO - {"message":"Model : '{model_data}' reloaded."}
[2025-01-24T21:23:18.748+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-24T21:23:18.776+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Train, task_id=update_model, execution_date=20250124T212301, start_date=20250124T212317, end_date=20250124T212318
[2025-01-24T21:23:18.836+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-24T21:23:18.855+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
